src/utils/mod.rs

```diff
@@ -1674,31 +1742,58 @@ pub enum PublicKey {
 
 /// Struct holding a pcr value.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct PcrValue {
+pub struct Digest {
```

Comment:
Nice! This is a really useful structure!

I was wondering if it's worth creating a generic `List` structure, but I'll put that idea in another issue :)